observation_portal/settings.py

```diff
@@ -283,6 +292,10 @@ def get_list_from_env(variable, default=None):
     }
 }
 
+## Duration constants for calculating overheads
+MAX_IPP_VALUE = os.getenv('MAX_IPP_VALUE', 2.0)  # the maximum allowed value of ipp
```

Comment:
Are there any issues when this setting is a string, like when it's read in from an env var?

Reply:
yeah, i will need to float() wrap these